doc/nrf/known_issues.rst

```diff
@@ -244,19 +245,19 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-10633: Incorrect data when using ACK-based Probing with Link Metrics
   When using the ACK-based Probing enhanced with Link Metrics, the Thread Information Element contains fixed data instead of the correct Link Metrics data for the acknowledged frame.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```

Comment:
still in progress, but should be fixed for 1.7

Reply:
@MarekPorwisz , any updates? Is this fixed now?

Reply:
Yes, it is fixed